udp.go

```diff
@@ -61,3 +61,13 @@ func SetMulticastSendAddrIPv4(s string) error {
 	ssdpAddrIPv4 = addr
 	return nil
 }
+
+// SetMulticastRecvAddrIPv4 updates multicast address where to receive packets.
+func SetMulticastRecvAddrIPv4(s string) error {
+	_, err := net.ResolveUDPAddr("udp4", s)
```

Comment:
why don't you use returned value from `net.ResolveUDPAddr`?

It was used in `SetMulticastSendAddrIPv4`.

Reply:
Actually we don't need net.ResolveUDPAddr since recvAddrIPv4 is a string not an net.UDPAddr which was the case of SetMulticastSendAddrIPv4. I just used it to catch an incorrect format on the address passed.